ember-primitives/src/components/accordion/accordion.gts

```diff
@@ -0,0 +1,150 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import { hash } from '@ember/helper';
+
+// temp
+//  https://github.com/tracked-tools/tracked-toolbox/issues/38
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { localCopy } from 'tracked-toolbox';
+
+import AccordionItem from './item';
+
+import type { WithBoundArgs } from '@glint/template';
+
+type AccordionSingleArgs = {
+  /**
+   * The type of accordion. If `single`, only one item can be selected at a time. If `multiple`, multiple items can be selected at a time.
+   */
+  type: 'single',
+  /**
+   * Whether the accordion is disabled.
+   */
+  disabled?: boolean;
+} & ({
+  /**
+   * The currently selected value. To be used in a controlled fashion in conjunction with `onValueChange`.
+   */
+  value: string;
+  /**
+   * A callback that is called when the selected value changes. To be used in a controlled fashion in conjunction with `value`.
+   */
+  onValueChange: (value: string | undefined) => void;
+  defaultValue?: never;
```

Comment:
what's default value in this arg group for? (can you add a jsdoc comment?)


also, is defaultValue needed when uncontrolled? I think it'd be good to type out in here the situation in whihc you'd want defaultValue, vs just value vs defaultValue with value

Reply:
(I see now that the `never` type is probably used for appropriate narrowing, saying something like "not available when X" would be appropriate here, and the rest of my comment can go on the active `defaultValue` jsdocs)

Reply:
Indeed! I used the `never` type so it cannot be provided when using the accordion in a controlled manner. I also added this to the JSDoc now